src/app/selectors.js

```diff
@@ -205,6 +205,20 @@ const countModelStatusGroups = groupedModelStatuses => {
 
 // ----- Exported functions
 
+/**
+  Checks state to see if the sidebar is collapsible.
+  Usage:
+    const isSidebarCollapsible = useSelector(isSidebarCollapsible);
+
+  @param {Object} state The application state.
+  @returns {Boolean} If the sidebar is collapsible.
+*/
+export const isSidebarCollapsible = state => {
```

Comment:
If you were to change this to support an enum this would be changed to something like `getSidebarVisibility`